unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -610,11 +1151,16 @@ displayTo =
         makeExample displayTo ["<filename>", "foo"]
           <> "prints a rendered version of the term `foo` to the given file."
     )
-    \case
-      file : (x : xs) ->
-        (x NE.:| xs)
-          & traverse parseHashQualifiedName
-          <&> Input.DisplayI (Input.FileLocation file)
+    $ \case
```

Comment:
We have the `LambdaCase` extension enabled, so we leave off these `$` where possible

Reply:
Whoops – there was an interim commit (8a95c5fe52f5f4395b21bacd542080ab5ff15616) that required adding those, and looks like I missed removing some in the follow-up.